### PR TITLE
[T4A4][T11-A1]Justin Wong

### PR DESCRIPTION
**Pull request by justinhw**

### CODE REVIEW

**Comment by JustinSYDE** [comment]
Ready to review.

File: src/seedu/addressbook/data/person/Printable.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
```
Comment (nirandiw) [inline src/seedu/addressbook/data/person/Printable.java:2]: Header comments are missing

File: src/seedu/addressbook/data/person/ReadOnlyPerson.java [hunk 2]
```diff
@@ -83,4 +84,17 @@ default String getAsTextHidePrivate() {
...
     }
+    
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+     default String getPrintableString(Printable... printables) {
+    	 String printableString = ""; 
+    	 
+    	 for (Printable printable : printables) {
+    		 printableString += printable.getPrintableString() + "\n"; // append a newline to the end of each printable
```
Comment (nirandiw) [inline src/seedu/addressbook/data/person/ReadOnlyPerson.java:95]: Was it not suppose to be comma separated?

**Review by nirandiw — COMMENTED** [review]
@JustinSYDE Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**